components/cam/src/physics/rrtmgp/radiation.F90

```diff
@@ -271,7 +281,8 @@ subroutine radiation_readnl(nlfile, dtime_in)
              '  Use average zenith angle:                           ',l5/, &
              '  Output spectrally resolved fluxes:                  ',l5/, &
              '  Do aerosol radiative calculations:                  ',l5/, &
-             '  Fixed solar consant (disabled with -1):             ',f10.4/ )
+             '  Fixed solar consant (disabled with -1):             ',f10.4/, &
+             '  Enable limiter warnings:                            ',l5/ )
```

Comment:
Likewise, this should read "Enable temperature warnings" for consistency with the renamed namelist variable.

Reply:
Done.